test_framework/test_verifier.py

```diff
@@ -0,0 +1,67 @@
+import os
+import tempfile
+import struct
+from subprocess import Popen, PIPE
+from nose.plugins.skip import Skip, SkipTest
+import ubpf.assembler
+import testdata
+VM = os.path.join(os.path.dirname(os.path.realpath(__file__)), "..", "vm", "test")
+
+def check_datafile(filename):
+    """
+    Given assembly source code and an expected result, run the eBPF program and
+    verify that the result matches.
+    """
+    data = testdata.read(filename)
+    if 'asm' not in data and 'raw' not in data:
+        raise SkipTest("no asm or raw section in datafile")
+    if 'result' not in data and 'verifier error' not in data:
+        raise SkipTest("no result or verifier error section in datafile")
```

Comment:
Shouldn't we also run on test cases that don't result in verifier errors, to check for false positives?